README.md

```diff
@@ -61,6 +62,12 @@ Or install it yourself as:
     Cronex::ExpressionDescriptor.new('30 2 * 2 1-5', {}, 'fr').description
     => À 2:30 AM, lundi à vendredi, seulement en février
 
+#### Timezones
+
+    Cronex::ExpressionDescriptor.new('0-10 11 * * *', timezone: 'America/Los_Angeles').description
```

Comment:
needs to be updated. e.g.
`Cronex::ExpressionDescriptor.new('0-10 11 * * *', {}, 'en', 'America/Los_Angeles').description`